osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -1876,6 +1876,34 @@
       },
       "coingecko_id": "evmos"
     },
+    {
+      "description": "The native EVM, governance, bridge, and staking token of Echelon",
+      "denom_units": [
+        {
+          "denom": "ibc/49C2B2C444B7C5F0066657A4DBF19D676E0D185FF721CFD3E14FA253BCB9BC04",
+          "exponent": 0,
+          "aliases": ["aechelon"]
+        },
+        {
+          "denom": "echelon",
+          "exponent": 18
+        }
+      ],
+      "base": "ibc/49C2B2C444B7C5F0066657A4DBF19D676E0D185FF721CFD3E14FA253BCB9BC04",
+      "name": "Echelon",
+      "display": "echelon",
+      "symbol": "ECH",
+      "ibc": {
+        "source_channel": "channel-8",
+        "dst_channel": "channel-262",
+        "source_denom": "aechelon"
+      },
+      "logo_URIs": {
+        "png": "https://app.ech.network/logo.png",
+        "svg": "https://app.ech.network/echelon.svg"
```

Comment:
Can you upload these images into an images subfolder in the chain registry

See example:  https://github.com/cosmos/chain-registry/tree/master/osmosis/images

Reply:
Yes, will do!